frontend/pendingTranslations.ftl

```diff
@@ -32,3 +32,6 @@ profile-maxed-aliases-with-phone-description = You’ve hit your { $limit }-mask
 profile-maxed-aliases-without-phone-description = You’ve hit your { $limit }-mask limit on your free { -brand-name-relay } account. Upgrade to { -brand-name-relay-premium } for unlimited email masks.
 profile-maxed-aliases-cta = Upgrade to { -brand-name-premium }
 profile-label-set-your-custom-domain-free-user = Get your own email domain with { -brand-name-premium }
+
+tooltip-email-domain-explanation-title-free = Get your own { -brand-name-relay } Email Domain
```

Comment:
L10n Q: Why is "Email Domain" capitalized?
If this was a header, wouldn't "Your Own" also be capitalized?

Reply:
I'm not sure, I can ask content, design or PM to make a decision on that. I normally yield to L10N team. I'll ask on Tuesday (after Labor day on Monday) as they're out of office at this moment.

Reply:
Looks like original `tooltip-email-domain-explanation-title` = About your unique { -brand-name-relay } Email Domain

Was using the same - I went ahead and updated based on your input.